<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StyleWidget</class>
 <widget class="QWidget" name="StyleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="1" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>445</width>
        <height>438</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QToolBox" name="styleEditor">
         <property name="styleSheet">
          <string notr="true">QToolBox::tab {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
    border-radius: 5px;
    color: black;
}
QToolBox::tab:selected { /* italicize selected tabs */
    font:bold;
    color: black;
}</string>
         </property>
         <property name="currentIndex">
          <number>5</number>
         </property>
         <property name="tabSpacing">
          <number>6</number>
         </property>
         <widget class="QWidget" name="WindowStyle">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>427</width>
            <height>231</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Window</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <property name="spacing">
            <number>3</number>
           </property>
           <item row="0" column="0">
            <widget class="window_subwindow_widget" name="windowStyleEditor" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="subwindowStyle">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>427</width>
            <height>231</height>
           </rect>
          </property>
          <attribute name="label">
           <string>SubWindow</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <property name="spacing">
            <number>3</number>
           </property>
           <item row="0" column="0">
            <widget class="window_subwindow_widget" name="subwindowStyleEditor" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="ButtonsStyle">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>427</width>
            <height>231</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Buttons</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <property name="spacing">
            <number>3</number>
           </property>
           <item row="0" column="0">
            <widget class="button_input_widget" name="buttonStyleEditor" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="inputStyles">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>427</width>
            <height>231</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Inputs</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <property name="spacing">
            <number>3</number>
           </property>
           <item row="0" column="0">
            <widget class="button_input_widget" name="inputStyleEditor" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="elementTextStyles">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>427</width>
            <height>231</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Elements Text</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_7">
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <property name="spacing">
            <number>3</number>
           </property>
           <item row="0" column="0">
            <widget class="text_style_editor_widget" name="elementTextStylesEditor" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="inputTextStyles">
          <attribute name="label">
           <string>Inputs Text</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_9">
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <property name="spacing">
            <number>3</number>
           </property>
           <item row="0" column="0">
            <widget class="text_style_editor_widget" name="inputTextStylesEditor" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="ListStyles">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>427</width>
            <height>231</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Misc.</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_8">
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <property name="spacing">
            <number>3</number>
           </property>
           <item row="0" column="0">
            <widget class="List_style_widget" name="ListStylesEditor" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_save_as">
       <property name="text">
        <string>Save as</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_apply">
       <property name="text">
        <string>apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Delete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="Styles">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>window_subwindow_widget</class>
   <extends>QWidget</extends>
   <header>window_subwindow_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>button_input_widget</class>
   <extends>QWidget</extends>
   <header>button_input_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>text_style_editor_widget</class>
   <extends>QWidget</extends>
   <header>text_style_editor_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>List_style_widget</class>
   <extends>QWidget</extends>
   <header>list_style_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
